### PR TITLE
Expose Satellite as source type in UI

### PR DESCRIPTION
**Pull request by rvsia**
@gtanzillo 

This PR exposes Satellite source type in UI.

Not sure which values we need to collect, which are required, validators, etc.

### CODE REVIEW

File: db/seeds/source_types.rb [hunk 1]
```diff
@@ -145,4 +145,39 @@ def update_or_create(attributes)
...
+        {
+          :component         => "text-field",
+          :name              => "authentication.authtype",
+          :hideField         => true,
+          :initializeOnMount => true,
+          :initialValue      => "receptor_node"
+        },
+        {
+          :component  => "text-field",
+          :name       => "authentication.username",
```
Comment (gtanzillo) [inline db/seeds/source_types.rb]: I'm thinking that this is not the userid but instead is an id the references the Satellite instance in the on prem env. I'll check on that.

Comment (agrare) [reply db/seeds/source_types.rb]: If the satellite ID is the unique identifier for the satellite we have `source.source_ref` for that

Comment (gtanzillo) [reply db/seeds/source_types.rb]: Thanks Adam, Richard and I are discussing it now. I'll find out for sure and post here

Comment (gtanzillo) [reply db/seeds/source_types.rb]: @rvsia @agrare I've confirmed that the Satellite ID that we need to store with the sources is a GUID that is used to identify an instance of a Satellite running on prem. So `source.source_ref` would be the place to save it.

**Pull request merged**